xforce.json

```diff
@@ -1,16 +1,16 @@
 {
     "appid": "653a4735-1eed-4cb9-98c4-b40408e60b22",
-    "name": "IBM XForce",
-    "description": "This app implements various 'investigative' actions on the 'IBM XForce' device",
+    "name": "IBM X-Force Exchange",
+    "description": "This app implements various 'investigative' actions on the 'IBM X-Force Exchange' device",
     "publisher": "AvantGarde Partners",
-    "package_name": "phantom_xforce",
+    "package_name": "phantom_ibmx-forceexchange",
     "type": "investigative",
-    "license": "Copyright (c) AvantGarde Partners 2017",
+    "license": "Copyright (c) IBM X-Force Exchange",
     "main_module": "xforce_connector.py",
     "app_version": "1.1.0",
     "utctime_updated": "2022-01-07T20:46:01.000000Z",
     "product_vendor": "IBM",
```

Comment:
Should this be updated to `IBM X-Force Exchange` as well?

Reply:
The vendor or the connector? Either way, I think currently it's fine. IBM is indeed still the vendor for the product

Reply:
@joshquerinphantom  Oops sorry I should have been more clear.
I'm referring to the `product_vendor` here.
Wondering if it should be updated from `IBM` -> `IBM X-Force Exchange`
I think you are right to keep it as `IBM` though. I will let @mattsayar-splunk  confirm

Reply:
Leaving it as IBM is good